nautobot/extras/tests/test_datasources.py

```diff
@@ -122,6 +122,7 @@ def tearDown(self):
             del sys.modules[f"{self.repo.slug}.jobs"]
         if f"{self.repo.slug}" in sys.modules:
             del sys.modules[f"{self.repo.slug}"]
+        super().tearDown()
```

Comment:
Why is this needed?

Reply:
General good practice now that we have a `tearDown()` in the base class - it shouldn't be strictly necessary in this specific case at this specific time but will protect us in the eventuality that we add more to the base class method. As far as I could tell with some quick greps, this was the only test case we have with a `tearDown()` of its own that wasn't already calling super.